generator/generator_tests/osm_type_test.cpp

```diff
@@ -1437,6 +1437,48 @@ UNIT_CLASS_TEST(TestWithClassificator, OsmType_Organic)
   }
 }
 
+UNIT_CLASS_TEST(TestWithClassificator, OsmType_Internet)
+{
+  {
+    Tags const tags = {
+      {"internet_access", "no"},
+      {"wifi", "no"},
+      {"amenity", "cafe"},
+    };
+
+    auto const params = GetFeatureBuilderParams(tags);
+
+    TEST_EQUAL(params.m_types.size(), 1, (params));
+    TEST(params.IsTypeExist(GetType({"amenity", "cafe"})), (params));
+  }
+
+  {
+    Tags const tags = {
+      {"internet_access", "wlan"},
+      {"office", "it"},
+    };
+
+    auto const params = GetFeatureBuilderParams(tags);
+
+    TEST_EQUAL(params.m_types.size(), 2, (params));
+    TEST(params.IsTypeExist(GetType({"office"})), (params));
+    TEST(params.IsTypeExist(GetType({"internet_access", "wlan"})), (params));
+  }
+
+  {
+    Tags const tags = {
+      {"wifi", "free"},
+      {"shop", "clothes"},
+    };
```

Comment:
maybe add a test for 
```
{"wifi", "free"},
{"internet_access", "yes"}
```
will it lead to types duplication?

and
```
{"wifi", "free"},
{"internet_access", "terminal"}
```
should lead to wlan type present